src/main/java/com/juniorstart/juniorstart/payload/ChangePasswordRequest.java

```diff
@@ -15,13 +16,14 @@
  * @since 1.0
  */
 @Getter
+@NoArgsConstructor
 public class ChangePasswordRequest implements InterfaceChangeRequest {
     @Email
```

Comment:
można się dowiedzieć czemu taka zmiana?

Reply:
Jeżeli nie dodasz bezparametrowego konstruktora to Ci MockMVC nie zadziała: https://stackoverflow.com/questions/63980929/mockmvc-junit-5-post-uuid-test

Reply:
chodziło mi z polami final, ale też to warto wiedzieć :)